### PR TITLE
Action notifications

### PR DESCRIPTION
**Pull request by ArkadiGiniApps**

### CODE REVIEW

**Comment by ArkadiGiniApps** [comment]
Some stuff for amir

File: GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift [hunk 1]
```diff
@@ -11,14 +11,15 @@ import UIKit
 protocol GAGalleryContainerBusinessLogic {
     
     var currentShareURL : URL? { get set }
-    var dataStore : [GAGalleryItem] { get set }
+    var dataStore   : [GAGalleryItem] { get set }
```
Comment (IdoMGini) [inline GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift]: align properties

File: GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift [hunk 2]
```diff
@@ -11,14 +11,15 @@ import UIKit
...
     func fetchItemInfo(_ request: GAGalleryContainer.FetchItemInfo.Request)
 }
 
 protocol GAGalleryContainerDataStore {
     
     var currentShareURL : URL? { get set }
     var dataStore : [GAGalleryItem] { get set }
+    var currentItem : GAGalleryItem? { get }
```
Comment (IdoMGini) [inline GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift]: align properties

File: GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift [hunk 3]
```diff
@@ -44,12 +47,30 @@ class GAGalleryContainerInteractor: GAGalleryContainerBusinessLogic, GAGalleryCo
         
         let index = request.itemIndex
         
-        guard index != NSNotFound else { return }
-        
+        guard index != NSNotFound	             else { return }
```
Comment (IdoMGini) [inline GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift]: align properties

File: GAGalleryKit/Classes/Utilities/GAShareManager.swift [hunk 4]
```diff
@@ -90,11 +90,11 @@ open class GAShareManager : NSObject {
         mailComposeViewController.mailComposeDelegate = GAShareManager.defaultsShareHelper
         
         viewController.present(mailComposeViewController, animated: true, completion: nil)
-        NotificationCenter.default.post(name: Notification.Name.GAGalleryShareToEmail, object: nil)
+        NotificationCenter.default.post(name: .GAGalleryShareToEmail, object: nil)
```
Comment (IdoMGini) [inline GAGalleryKit/Classes/Utilities/GAShareManager.swift]: you forgot to add the shareItem to the post object

File: GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift (hunk 1 above)
Comment (amirzucker) [reply GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift]: Done

File: GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift (hunk 2 above)
Comment (amirzucker) [reply GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift]: Done

File: GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift (hunk 3 above)
Comment (amirzucker) [reply GAGalleryKit/Classes/Modules/Gallery/Container/GAGalleryContainerInteractor.swift]: there is nothing to align to.

File: GAGalleryKit/Classes/Utilities/GAShareManager.swift (hunk 4 above)
Comment (amirzucker) [reply GAGalleryKit/Classes/Utilities/GAShareManager.swift]: Done.

**Pull request merged**